distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -847,78 +844,6 @@ type Table
             parse_problem_builder.attach_problems_before on_problems <|
                 Table.new new_columns
 
-    ## Replaces the first, or all occurrences of `term` with `new_text` in each
-       text row of selected columns.
-       If `term` is empty, the function returns the table unchanged.
-
-       This method follows the exact replacement semantics of the
-       `Text.replace` method.
```

Comment:
Why is this being removed?

I assume because of redundancy with `Column.replace`?

Personally I like the directness of `Table` operations - like we have `Table.cast column_selector` as a counterpart to `Column.cast`.

IMHO it makes them easier to use in the GUI as I just need one node:
```
table.replace_text "Column_A" "-" "_"
```
instead of a whole chain:
```
c = table.at "Column_A"
c1 = c.replace "-" "_"
table.set c1
```

Although you are using IDE much more than me, so I rely on your judgement on this, but just adding my 2 cents.

Reply:
We've opted for operations on a single column being a column operation. 

You can always map over a set of columns and yes agree the `at -> op -> set` still needs some thought more generally.

Reply:
Does it mean we should remove `Table.cast` too from our plans? (for now I just got a prototype)

Although I'd be for keeping it, it makes writing tests more concise:
![image](https://user-images.githubusercontent.com/1436948/233324770-9b07060a-9998-4ae7-bcb3-9e11412e150c.png)